docs/tools.rst

```diff
@@ -57,6 +57,15 @@ when the feature branch lands, so you will need to update this yourself.
 Please read `Useful Github Patterns <http://blog.quickpeople.co.uk/2013/07/10/useful-github-patterns/>`_
 to see ways of working with branches and pull requests that we like.
 
+Hubflow
+-------
+
+Hubflow is an adapted version of Gitflow, specifically tailored for use with Github.
```

Comment:
Nitpick -- `HubFlow`, `GitFlow` and `GitHub` should probably all have capitals in the middle when used as proper nouns.